src/sentry/web/frontend/base.py

```diff
@@ -233,6 +236,19 @@ def get_team_list(self, user, organization):
             with_projects=True,
         )
 
+    def create_audit_entry(self, request, **kwargs):
+        entry = AuditLogEntry.objects.create(
+            actor=request.user if request.user.is_authenticated() else None,
+            ip_address=request.META['REMOTE_ADDR'],
+            **kwargs
+        )
+        logger.info(
+            name='sentry.audit.entry',
+            event=entry.get_event_display(),
+            actor_id=entry.actor_id,
```

Comment:
In reality, yes, but since we now have assumed that `actor` is a nullable field, `request.user.id` will error out when `request.user.is_authenticated` does not return true.